streaming/src/main/scala/io/gearpump/streaming/task/Subscription.scala

```diff
@@ -74,17 +75,29 @@ class Subscription(
     transport.transport(ackRequest, allTasks: _*)
   }
 
+  def sendMessage(msg: Message): Int = {
+    partitioner match {
+      case up: UnicastPartitioner =>
```

Comment:
Can we avoid do the pattern match every time?

Will there be possible performance degradtion?